src/CoreWCF.RabbitMQ/tests/CoreWCF.RabbitMQ.Tests.csproj

```diff
@@ -22,14 +27,9 @@
     <PackageReference Include="RabbitMQ.Client" Version="6.4.0" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <ProjectReference Include="..\..\CoreWCF.RabbitMQ.Client\src\CoreWCF.RabbitMQ.Client.csproj" />
```

Comment:
Your conditions only allow for net6.0. You should check against net472 otherwise you don't get anything executed on .net7.0. So this condition would be != 'net472', and the one above would be == 'net472'.